src/ot_orchestration/operators/gce.py

```diff
@@ -572,17 +563,18 @@ def copy_machine_logs(self) -> None:
             page_size=1000,
         )
         for entry in entries:
-            self.log.log(level=logging.INFO, msg=entry.payload.get("message", ""))
+            self.log.info(entry.payload.get("message", "Empty log message"))
```

Comment:
What is the reason of this logs?

Reply:
Some times google cloud generates empty log messages (maybe if there is a newline somewhere). It should not happen in PIS, but just in case.